op-conductor/flags/flags.go

```diff
@@ -0,0 +1,77 @@
+package flags
+
+import (
+	"fmt"
+
+	"github.com/urfave/cli/v2"
+
+	opservice "github.com/ethereum-optimism/optimism/op-service"
+	opflags "github.com/ethereum-optimism/optimism/op-service/flags"
+	oplog "github.com/ethereum-optimism/optimism/op-service/log"
+	opmetrics "github.com/ethereum-optimism/optimism/op-service/metrics"
+	oppprof "github.com/ethereum-optimism/optimism/op-service/pprof"
+	oprpc "github.com/ethereum-optimism/optimism/op-service/rpc"
+)
+
+const EnvVarPrefix = "OP_CONDUCTOR"
+
+var (
```

Comment:
Could you add a flags_test.go file? Right now we have a set of standard flag tests, but they are all copy/paste: https://github.com/ethereum-optimism/optimism/blob/develop/op-batcher/flags/flags_test.go

Reply:
tests added